steps/check/step.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+apk add --update nodejs npm jq
+npm install -g linkinator
+URL="$(ni get -p '{.websiteURL}')"
+echo "${URL}"
+LINKS_ARRAY="$(linkinator "${URL}" --recurse --format JSON)"
+BAD_LINKS=$( jq '[.links[] | select(.status == 404) | .url]' <<<"${LINKS_ARRAY}" )
+LINKS_COUNT=$( jq 'length' <<<"${BAD_LINKS}" )
+echo "Bad links found ${LINKS_COUNT}"
```

Comment:
The statements you're printing here could use another pass ("Checking website ${URL}...", "${LINKS_COUNT} bad link(s) found" maybe?)